src/du/du.rs

```diff
@@ -62,16 +64,32 @@ impl Stat {
             size: metadata.len(),
             blocks: metadata.blocks() as u64,
             nlink: metadata.nlink() as u64,
+            inode: metadata.ino() as u64,
             created: metadata.mtime() as u64,
             accessed: metadata.atime() as u64,
             modified: metadata.mtime() as u64,
         })
     }
 }
 
+#[cfg(target_os = "macos")]
+fn get_default_blocks() -> u64 {
+    512
```

Comment:
I just ran `gdu` on my Mac, and it seems to be using 1024-byte blocks AFAICT.  The default `du` (inherited from BSD) does seem to use 512-byte blocks though.  As we are trying to mimic the behavior of the GNU version, 1024 seems to make the most sense by default.

I have found [this](https://www.gnu.org/software/coreutils/manual/html_node/Block-size.html) though (which I was not aware of before), so perhaps you can implement this?  It sounds the GNU version isn't technically correct according to POSIX, unlike the Mac version, which is the source of the discrepancy.

Reply:
I'm happy to make it read the POSIXLY_CORRECT flag. But I don't think we've got to the root cause of confusion yet. 

env | grep POS
env | grep BLOCK
returns nothing on my macbook or my linux server.

Going to do some digging.

Reply:
Yes looks like you are right with du / gdu. 

Strangely I'd started implementing the BLOCK_SIZE environment variables on a branch as I'd noticed they were missing. I'll incorporate that work.